src/Core/src/Handlers/Entry/EntryHandler.Android.cs

```diff
@@ -1,37 +1,60 @@
 ﻿using System;
 using Android.Content.Res;
+using Android.Graphics.Drawables;
 using Android.Text;
+using Android.Views;
 using AndroidX.AppCompat.Widget;
+using AndroidX.Core.Content;
 using Microsoft.Extensions.DependencyInjection;
+using static Android.Views.View;
 
 namespace Microsoft.Maui.Handlers
 {
 	public partial class EntryHandler : AbstractViewHandler<IEntry, AppCompatEditText>
 	{
 		TextWatcher Watcher { get; } = new TextWatcher();
+		EntryTouchListener TouchListener { get; } = new EntryTouchListener();
+		EntryFocusChangeListener FocusChangeListener { get; } = new EntryFocusChangeListener();
 
 		static ColorStateList? DefaultTextColors { get; set; }
+		static Drawable? ClearButtonDrawable { get; set; }
 
 		protected override AppCompatEditText CreateNativeView()
 		{
 			return new AppCompatEditText(Context);
 		}
 
+		// Returns the default 'X' char drawable in the AppCompatEditText.
+		protected virtual Drawable GetClearButtonDrawable()
+			=> ContextCompat.GetDrawable(Context, Resource.Drawable.abc_ic_clear_material);
```

Comment:
@PureWeen Just pointing this out. Not sure what the official way to override things are like this. Should it be some mapper or some other registration that can be injected without requiring an override in a new type? We obviously need to be careful with having a bajillion mapper functions for everything. Maybe we need to clarify where the line of mapper/override/parameter is.